.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules/
 public/
 resources/
+.idea
```

Comment:
why this change needed?

Reply:
when we use either `goland` or `vscode` IDE these files will be created and that should be ignored.